modes/company/evil-collection-company.el

```diff
@@ -47,11 +47,22 @@ be set through custom or before evil-collection loads."
   :group 'evil-collection-company
   :type 'boolean)
 
+(defcustom evil-collection-company-supported-states `(insert replace emacs)
```

Comment:
Use a simple quote instead of a backquote since you are not unquoting.

Reply:
Done